static/app/utils/analytics/issueAnalyticsEvents.tsx

```diff
@@ -215,7 +215,7 @@ export type IssueEventParameters = {
   resolve_issue: {release: string};
   'settings.inbound_filter_updated': {
     filter: string;
-    new_state: FieldValue;
+    new_state: FieldValue | string[];
```

Comment:
@roggenkemper we can't do lists of strings for analytics, it won't go to Amplitude correctly